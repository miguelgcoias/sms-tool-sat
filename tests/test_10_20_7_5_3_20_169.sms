10
16 4 20 3 2 1 1
15 4 23 1 4
16 1 17 1 1
1 7 15 2 3 4
11 6 17 3 2 2 2
2 7 16 3 1 2 4
0 1 2 1 1
5 2 7 1 2
10 5 20 1 5
0 2 4 2 1 1
6 7 9 8 6 4 10
6 7 5 6 4 8 9
5 4 7 9 8 6
1 8
4 8 6 7 10
3 8 4 7
0
1 7
3 6 7 8
1 7
