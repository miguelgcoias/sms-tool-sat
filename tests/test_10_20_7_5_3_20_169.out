4
4 7 10
7 0
8 5
10 1 3
